assets/js/apicalls.js

```diff
@@ -0,0 +1,82 @@
+
+
+const omdbAPIkey = "5af38ba4";
+var test = document.getElementById('root');
+
+function fetchAllGhibliMovies() {
+  apicall();
+  buildOMDBApiCall();
+  var request = new XMLHttpRequest();
+  var container = document.createElement('div');
+  container.setAttribute('class', 'container');
+  test.appendChild(container);
+
+    request.open('GET', 'https://ghibliapi.herokuapp.com/films', true);
```

Comment:
See comment above. Code should be refactored so that it returns an array of Ghibli movies using the data received from the API call.